src/array.cr

```diff
@@ -1930,6 +1930,17 @@ class Array(T)
     self
   end
 
+  # Prints a nicely readable and concise string representation of this array
+  # to *io*.
+  #
+  # The result resembles an array literal but it does not necessarily compile.
+  #
+  # It invokes `#inspect(io)` on each element to avoid ambiguity.
+  #
+  # ```
+  # ["one", "two, three"].inspect # => ["one", "two, three"]
+  # #                               vs [one, two, three]
```

Comment:
I'm not sure adding and example of what is not returned clarifies something.

Reply:
Okay, I'll remove the example.